Remove unused React imports from 6 files

Leone2022 opened the pull request:
- ConfirmPopup.js
- CustomCheckBox.js
- CustomSelect.js
- GlobalSideBar.js
- PageBreadCrumb.js
- Utils.js

Modern React no longer requires explicit React imports.

Changes
- Removed unused `import React` from ConfirmPopup.js
- Removed unused `import React` from CustomCheckBox.js
- Removed unused `import React` from CustomSelect.js
- Removed unused `import React` from GlobalSideBar.js
- Removed unused `import React` from PageBreadCrumb.js
- Removed unused `import React` from Utils.js

 Testing
- Code builds successfully
- No functionality affected (React imports are optional in modern React)

Leone2022 committed:
Remove unused React imports from 6 files

- ConfirmPopup.js
- CustomCheckBox.js
- CustomSelect.js
- GlobalSideBar.js
- PageBreadCrumb.js
- Utils.js

Modern React no longer requires explicit React imports.

Fixes part of #1554

Agaba-derrick commented on ed React imports from 6 files:
```diff
@@ -0,0 +1,61 @@
+[1mdiff --git a/frontend/src/components/common/ConfirmPopup.js b/frontend/src/components/common/ConfirmPopup.js[m
```
@Leone2022 thanks for the PR, are you able to format your code before pushing ?
ie  cd frontend 
then run npm install prettier
and  then run npm run format

Agaba-derrick commented:
> * ConfirmPopup.js
> 
>     * CustomCheckBox.js
> 
>     * CustomSelect.js
> 
>     * GlobalSideBar.js
> 
>     * PageBreadCrumb.js
> 
>     * Utils.js
> 
> 
> Modern React no longer requires explicit React imports.
> 
> Changes
> 
>     * Removed unused `import React` from ConfirmPopup.js
> 
>     * Removed unused `import React` from CustomCheckBox.js
> 
>     * Removed unused `import React` from CustomSelect.js
> 
>     * Removed unused `import React` from GlobalSideBar.js
> 
>     * Removed unused `import React` from PageBreadCrumb.js
> 
>     * Removed unused `import React` from Utils.js
> 
> 
> Testing
> 
>     * Code builds successfully
> 
>     * No functionality affected (React imports are optional in modern React)

Use the OpenELIS Pull Request Template to write your commit Description, Remember to tag the issue number you are working on  ie Fixes, Closes, Solves  followed by the Issue number 
ie

Leone2022 committed:
Migrate Role entity from HBM XML to JPA annotations

- Added JPA annotations (@Entity, @Table, @Column, etc.) to Role.java
- Disabled Role.hbm.xml mapping file by renaming to .disabled
- Verified build success with Maven compile
- All fields properly annotated including custom types

This is the first entity migrated as part of issue #2276

mozzy11 commented:
Thanks @Leone2022 .
Can you resolve the merge conflicts ??
Also looks like your Pr contains more changes beyond what you intended to do .
Can you only have changes for this specifc issuue?

mozzy11 committed:
Merge branch 'develop' into remove-unused-imports

Leone2022 committed:
Merge branch 'develop' into remove-unused-imports
